### ISSUE TITLE
无法加载媒体，因为服务器或网络出现故障或者不支持该格式

### ISSUE THREAD

**Issue by zs910208565** [You2php/you2php#214]
大部分视频显示

无法加载媒体，因为服务器或网络出现故障或者不支持该格式     
有解决方法吗？ #

**Comment by zs910208565** [comment]
@You2php
@asiacny
@liruqi
@dythebs

**Comment by FaArIsH** [comment]
Yes there is a solution. You can edit the lib.php and add YouTube embed link with miniproxy.php or you can use video just YouTube tech

I simple used this https://github.com/FaArIsH/you2php/commit/cd3235a2498e968a49c565057a4610801142494a

**Comment by zs910208565** [comment]
反馈，用的测试lib.php

4k可以播放，普通界面播放那无视频
![screenshot_2018-10-01-08-12-45-782_com android chrome](https://user-images.githubusercontent.com/34202810/46264679-fe0ba900-c551-11e8-94b1-22d92332f05f.png)
![screenshot_2018-10-01-08-12-30-571_com android chrome](https://user-images.githubusercontent.com/34202810/46264690-124fa600-c552-11e8-9347-a1d40df29d72.png)

**Comment by zs910208565** [comment]
@FaArIsH

**Comment by zxq2233** [comment]
@zs910208565 收到，目前youtube接口升级了，导致某些视频解析失效了。等待解析库作者跟进升级，本程序将在后续更新

**Comment by zs910208565** [comment]
好的👌谢谢

**Comment by FaArIsH** [comment]
@zxq2233 Lool it works
![screenshot_2018-10-02-13-26-36-500_com android chrome](https://user-images.githubusercontent.com/10223927/46337799-e878ab00-c647-11e8-9b52-699357059b05.png)

**Comment by johnwmail** [comment]
> @zs910208565 收到，目前youtube接口升级了，导致某些视频解析失效了。等待解析库作者跟进升级，本程序将在后续更新

請問那裹是解析库作者? 能結個url嗎?
我也有這問題.
